lib/osf-components/addon/components/form-controls/template.hbs

```diff
@@ -39,7 +39,7 @@
     )
     select=(
         component
-        'validated-input/power-select'
+        'form-controls/power-select'
```

Comment:
More of a question, but if we're already calling `FormControls::Select` should we also call this component `select` instead of `power-select` to keep it consistent?